Tests/AuthServiceTests/Validators/UserEmailPasswordValidatorTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AuthenticationService.Validators;
+using DTO;
+using FluentValidation;
+using NUnit.Framework;
+using System.Linq;
+
+namespace AuthenticationServiceTests.Validators
+{
+    public class UserEmailPasswordValidatorTests
+    {
+        private UserEmailPassword emailPasswordEmpty = new UserEmailPassword(string.Empty, string.Empty);
+
+        private UserEmailPassword emailEmptyPasswordOk = new UserEmailPassword(string.Empty, "xxxxx");
+
+        private UserEmailPassword passwordEmptyEmailOk = new UserEmailPassword("example@gmail.com", string.Empty);
+
+        private UserEmailPassword emailInvalidFormat = new UserEmailPassword("ololo", "xxxxx");
```

Comment:
Why not emailInvalidFormatPasswordOk? Don't you have to write in the same style? 
If you check both of fields, check it everywhere. Oh you have to split it to 2 differents examples
And the same question to lines below

Reply:
Might have missed that